docs/manual/java/gettingstarted/GettingStartedMaven.md

```diff
@@ -67,18 +67,18 @@ $ mvn lagom:runAll
 (Services started, press enter to stop and go back to the console...)
 ```
 
-You can verify that the services are indeed up and running by invoking one of its endpoints, e.g:
+You can verify that the services are indeed up and running by invoking one of its endpoints from any HTTP client. For example, on Windows you might use Postman. On Linux, you might use curl:
```

Comment:
I wonder if we should tell people to just load it in their browser. We don't need to send Windows users away to install Postman just for this, and users who are already savvy with an HTTP client will probably know they could use the one they're familiar with.